src/signals/incident/components/IncidentForm/index.tsx

```diff
@@ -139,18 +151,16 @@ const IncidentForm = forwardRef<any, any>(
         if (next) {
           if (prevState.current.loading) {
             next()
```

Comment:
when submitting from the first to the second step, the form will be in a loading state. We want to add the stepsCompletedCount to the next step, step 2, cause thats the first unvalidated/invalid form step. In stepbystepnav the second label will be bold with a big blue ball.